virtual_rainforest/core/__init__.py

```diff
@@ -0,0 +1,67 @@
+from virtual_rainforest.core.config import register_schema
+
+
+# This is definitely a bit questionable, its basically here to make sure that the plants
+# module is imported somewhere when tests are run, this ensures that the plant schema is
+# added to testing. I don't think this is the best or right way to do something like
+# this, but couldn't come up with a better way to do it
+# TODO - Think of a better solution for this
+def load_plants_schema():
+    """Imports the plants module schema."""
+    from virtual_rainforest.plants import schema
+
+    return schema
+
+
+load_plants_schema()
+
+
+@register_schema("core")
+def schema():
```

Comment:
I really encourage you to use `Path` from [`pathlib`](https://docs.python.org/3/library/pathlib.html) instead of the functionality in `os` for path related operations. It is really the best practice nowadays and the preferred approach in the community. The above will look like:

```python
from pathlib import Path

config_schema = Path(__file__).parent.resolve() / "plant_schema.json"
```
which I hope you agree is more readable and cleaner. It's `glob` capabilities are also more easy to deal with for pattern matching filenames.

And yes, JSON file in the same directory than the relevant module.

Reply:
I very much like that `/` operator!